v1/failing.txt

```diff
@@ -1,10 +1,2 @@
-TestUnmarshal
-TestUnmarshal/#106
-TestNilMarshal
-TestNilMarshal/#08
-TestNilMarshal/#11
-TestNilMarshalerTextMapKey
 TestStringOption
-TestStringOption/Unmarshal/Deep/v1
-TestPointerReceiver
-TestPointerReceiver/Marshal/v1
+TestStringOption/Unmarshal/Deep/v1
```

Comment:
The only remaining cause of difference is that v1 inconsistently treats unmarshal errors as lazy (i.e., it continues processing).

Reply:
Amazing work on this Joe!!